deps.ts

```diff
@@ -0,0 +1,5 @@
+export { Drash } from "https://deno.land/x/drash@v1.0.5/mod.ts";
```

Comment:
Drash and ServerRequest can be exported from this repo's root deps.ts file. I know there isn't one yet, so you'll be creator of it :). We don't want the potential to have middleware that uses different Drash versions, so that's why it should go in the root deps.ts file. When we bump Drash's version, we'll make sure all middleware works for the same Drash version. If a middleware doesn't work, then we'll have to debug before we make a new release.

Also, this could change in the future. This is just what we're going with right now and we'll work out any issues with this workflow as time goes on.